BuildLoader.py

```diff
@@ -88,11 +88,15 @@ def prep_env ():
         print("Unsupported operating system !")
         sys.exit(1)
 
+    if 'SLIMBOOT_KEY_DIR' not in os.environ:

```

Comment:
I am trying to understand why is this needed?

Reply:
While slimboot build, we could have per-checks for required env.
As we use key_id's it would be to add check as pre-build. Adding help note to generate keys need to be added here.

In standlaone tools + Key id - check in singlesing.py would be used.